mod/groups/actions/groups/membership/join.php

```diff
@@ -43,7 +43,11 @@
 		} else {
 			register_error(elgg_echo("groups:cantjoin"));
 		}
-	} else {
+	} else if(check_entity_relationship($user->guid,'membership_request',$group->guid)) {
```

Comment:
- space after `if` (not a function call)
- space after each `,` in argument list
- `} else {`, not Drupal style